src/components/FeedbackForm/index.js

```diff
@@ -11,6 +11,7 @@ export const FeedbackForm = () => {
     value: '',
     status: false
   });
+  const [doneMsg, setDoneMsg] = useState({ value: '', status: false });
```

Comment:
`doneMessage` and `dropdownValue` and `inputValue` and `length` need to change  ---  consider these rules.
- the status is already _false_ if the state is empty or null and it becomes _true_ if it's not empty. so remove the status from (dropDownValue and doneMsg) state
- error messages should be in a separate state so, remove them from (length and inputValue) and make another state for every error message

**when you finish the above requerments you should have just one state as an object it's `length` state**

and finnaly handle the code below to work with it

Reply:
For doneMessage :-

- status for displaying div or not but value for Msg whether it's done or no

Reply:
The SOLID principles are a set of software design principles, the first principle is 

**S - Single Responsibility Principle**
- Any function must be responsible for doing only ONE thing. Only one potential change in the software’s specification should be able to affect the specification of the class.

implement it on the react.js state :arrow_up: